content/docs/console/accounts/saml/_index.md

```diff
@@ -121,3 +121,15 @@ If your IdP does not support emailAddress Name ID Format identifier, but support
 ### The IdP signing certificate expired and I can't login to Pulumi anymore
 
 Please [contact us](https://www.pulumi.com/about/#contact-us) for assistance with getting the IdP metadata XML updated.
+
+### An SSO binding was not found in the XML. Please contact your SSO provider.
+
+This error occurs when the metadata XML you are trying to save does not have any `<SingleSignOnService>` elements under the `<IDPSSODescriptor>`. The `<SingleSignOnService>` is used by the Pulumi Console to determine the authentication mechanism supported by the IDP. [Learn more](https://en.wikipedia.org/wiki/SAML_2.0#SAML_2.0_Bindings) about SAML 2.0 bindings. You must contact your IdP support or your system admin to fix the metadata XML.
```

Comment:
This is great to have! Even just having it here will help with Google searching.

@bermudezmt is considering having a "common errors" index. This would be good to add to that once we have it.